modules/understanding-upgrade-channels.adoc

```diff
@@ -1,39 +1,17 @@
 // Module included in the following assemblies:
 //
-// * updating/updating-cluster.adoc
-// * updating/updating-cluster-between-minor.adoc
+// * updating/updating-cluster-within-minor.adoc
 // * updating/updating-cluster-cli.adoc
 // * updating/updating-cluster-rhel-compute.adoc
 // * updating/updating-disconnected-cluster.adoc
 
 [id="understanding-upgrade-channels_{context}"]
-= {product-title} upgrade channels and releases
```

Comment:
Is this just removed because it's so similar to the assembly title?

Reply:
@jeana-redhat Mostly yes. There is a new PR coming with an overhaul to this section (probably later this month) that'll put an overview title back in.

Reply:
@jeana-redhat Actually, I'm going to add a title back in here. Maybe something like. "Upgrade channels and releases" for now. I can change it later. But I realized, I need that heading to be able to link to it from other sections.